www/templates/goals/leaderboard-modal.html

```diff
@@ -109,10 +109,10 @@ <h1 class = "title">Leaderboard</h1>
       <div class="list" style="background-color: #ededed;">
         <div ng-repeat="user in usersUnderTopThree" 
            class="card item-thumbnail-left-avatar leaderboard-card" id="avatar" style="padding-top: 10px !important;height: 117px !important;">
-          <figure class="herobox background_{{user.preferences.background}} true ? &quot;isUser&quot; : &quot;&quot;} undefined ? &quot;minimal&quot; : &quot;&quot;}" ng-class="{hasPet: (true &amp;&amp; user.items.currentPet), hasMount: (true &amp;&amp; user.items.currentMount), noBackgroundImage: !user.preferences.background, &quot;cast-target&quot;: applyingAction, isLeader: party.leader==user._id}" style="top: -10px !important; left: 0px !important;width: 0px;">
+          <figure class="herobox background_{{user.preferences.background}} true ? &quot;isUser&quot; : &quot;&quot;} undefined ? &quot;minimal&quot; : &quot;&quot;}" ng-class="{hasPet: (true &amp;&amp; user.items.currentPet), noBackgroundImage: !user.preferences.background, &quot;cast-target&quot;: applyingAction, isLeader: party.leader==user._id}" style="top: -10px !important; left: 0px !important;width: 0px;">
```

Comment:
Yes, this is for the all the users not in top 3 position. The card is to small to include the avatar, the pet and the mount so this version only displays the avatar and the pet.